src/libs/ReportActionsUtils.js

```diff
@@ -547,6 +547,20 @@ function getIOUReportIDFromReportActionPreview(reportAction) {
     return lodashGet(reportAction, 'originalMessage.linkedReportID', '');
 }
 
+/**
+ * Find the reportAction whose childReportID is taskReportID in reportActions with reportID
+ *
+ * @param {String} taskReportID
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getParentReportActionForTask(taskReportID, reportID) {
+    return _.find(
+        allReportActions[reportID],
+        (reportAction) => reportAction && `${reportAction.childReportID}` === `${taskReportID}` && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT,
```

Comment:
Is there a reason that we need to filter on `actionName` as well?

Reply:
When we create a task, related action is added as comment to the assignee report. It makes sense to add this filtering, I think

Reply:
Can we remove that filter? I don't think we will find two reports actions in same report having the same child

Reply:
okay